kvmagent/kvmagent/plugins/host_plugin.py

```
@@ -1355,6 +1355,16 @@ def fact(self, req):
         qemu_img_version = qemu_img_version.strip('\t\r\n ,')
         ipV4Addrs = [chunk.address for chunk in [x for x in iproute.query_addresses(ip_version=4) if
                                          x.address != '127.0.0.1' and not x.ifname.endswith('zs')]]
+        # IPv6 支持：使用 ipaddress 标准库过滤链路本地地址（RFC 4291 fe80::/10），去掉 zone id
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)